src/common/modals/InstanceManager/Screenshots.js

```diff
@@ -0,0 +1,594 @@
+/* eslint-disable */
+import React, { useEffect, useState, useCallback, useRef } from 'react';
+import { promises as fs, watch, createReadStream } from 'fs';
+import { clipboard, ipcRenderer } from 'electron';
+import fse from 'fs-extra';
+import path from 'path';
+import base64 from 'base64-stream';
+import getStream from 'get-stream';
+import styled from 'styled-components';
+import makeDir from 'make-dir';
+import { Checkbox } from 'antd';
+import { useSelector, useDispatch } from 'react-redux';
+import groupBy from 'lodash/groupBy';
+import isEqual from 'lodash/isEqual';
+import sortBy from 'lodash/sortBy';
+import {
+  ContextMenuTrigger,
+  ContextMenu,
+  MenuItem,
+  hideMenu
+} from 'react-contextmenu';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faTrash,
+  faCopy,
+  faLink,
+  faFolder
+} from '@fortawesome/free-solid-svg-icons';
+import { _getInstancesPath } from '../../utils/selectors';
+import { openModal } from '../../reducers/modals/actions';
+import { imgurPost } from '../../api';
+
+const getScreenshots = async screenshotsPath => {
+  const files = await fs.readdir(screenshotsPath);
+  const screenshots = [];
+  try {
+    await Promise.all(
+      files.map(async element => {
+        const stats = await fs.stat(path.join(screenshotsPath, element));
+        const fileBirthdate = new Date(stats.birthtimeMs);
+        const timeDiff = Date.now() - fileBirthdate;
+        const days = parseInt(Math.floor(timeDiff / 1000) / 60 / 60 / 24, 10);
+        screenshots.push({
+          name: element,
+          days,
+          timestamp: fileBirthdate,
+          size: stats.size
+        });
+      })
+    );
+    return screenshots.sort((a, b) => a.timestamp - b.timestamp);
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+const getImgurLink = async (imagePath, fileSize, setProgressUpdate) => {
+  const updateProgress = progressEvent => {
+    setProgressUpdate(
+      Math.round((progressEvent.loaded * 100) / progressEvent.total)
+    );
+  };
+
+  const imageReadStream = createReadStream(imagePath);
+  const encodedData = new base64.Base64Encode();
+  const b64s = imageReadStream.pipe(encodedData);
+  const base64String = await getStream(b64s);
+
+  if (fileSize < 10485760) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The numeric literal '10485760' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5464387)